client/src/KudosRow.jsx

```diff
@@ -1,16 +1,44 @@
+// KudosRow.jsx
+import { useEffect, useState } from "react";
 import "./KudosRow.css";
 import KudosCard from "./KudosCard";
-const KudosRow = () => {
+
+const KudosRow = ({ searchTerm }) => {
+  const [boardInfo, setBoardInfo] = useState([]);
+
+  const populateCards = async () => {
+    try {
+      const resp = await fetch("http://localhost:3000/kudos/board");
```

Comment:
Set the URL in a .env or config file

Reply:
I created env.local and added to gitignore is that okay?